docs/sample-v2-addon/rollup.config.mjs

```diff
@@ -19,15 +19,15 @@ export default {
     // These are the modules that users should be able to import from your
     // addon. Anything not listed here may get optimized away.
     addon.publicEntrypoints([
-      'components/**/!(*.css.d).js',
+      'components/**/*.js',
```

Comment:
By temporarily committing the `dist` folder, I checked that:

- With the patch and the wildcard pattern `components/**/*.js`, running the `build` script results in the same `dist`.